src/v1/list_joined_rooms.rs

```diff
@@ -0,0 +1,44 @@
+//! [GET /_synapse/admin/v1/users/:user_id/joined_rooms](https://github.com/matrix-org/synapse/blob/master/docs/admin_api/user_admin_api.rst#list-room-memberships-of-an-user)
+
+use js_int::UInt;
+
+use ruma::{api::ruma_api, identifiers::UserId};
```

Comment:
I missed this in previous reviews but most types from `ruma-identifiers` are also re-exported at the crate root of ruma.

```suggestion
use js_int::UInt;
use ruma::{api::ruma_api, UserId};
```